src/components/LayerUI.tsx

```diff
@@ -382,7 +383,29 @@ const LayerUI = ({
 }: LayerUIProps) => {
   const isMobile = useIsMobile();
 
-  const renderExportDialog = () => {
+  const renderJSONExportDialog = () => {
+    if (!UIOptions.canvasActions.export) {
```

Comment:
we will need to rename this to `exportToJSON`, and for image `exportIToImage`, this can be done in the next pr.